po/es.po

```diff
@@ -0,0 +1,706 @@
+msgstr "Nada"
+
+#: src/info.rs:30 src/query.rs:51
+msgid "package '{}' was not found"
+msgstr "no se pudo encontrar paquete '{}'"
+
+#: src/info.rs:57 src/info.rs:96
+msgid "Name"
+msgstr "Nombre"
+
+#: src/info.rs:58 src/info.rs:97
+msgid "Version"
+msgstr "Versión"
+
+#: src/info.rs:59 src/info.rs:98
+msgid "Description"
+msgstr "Descripción"
+
+#: src/info.rs:60 src/info.rs:106
+msgid "Groups"
+msgstr "Grupos"
+
+#: src/info.rs:61 src/info.rs:107
+msgid "Licenses"
+msgstr "Licencias"
+
+#: src/info.rs:62 src/info.rs:108
+msgid "Provides"
+msgstr "Provee"
+
+#: src/info.rs:63 src/info.rs:109
+msgid "Depends On"
+msgstr "Depende De"
+
+#: src/info.rs:64 src/info.rs:110
+msgid "Make Deps"
+msgstr "Construir Dependencias"
+
+#: src/info.rs:65 src/info.rs:111
+msgid "Check Deps"
+msgstr "Comprobar Dependencias"
+
+#: src/info.rs:66 src/info.rs:113
+msgid "Conflicts With"
+msgstr "En Conflicto Con"
+
+#: src/info.rs:67 src/info.rs:114
+msgid "Maintainer"
+msgstr "Mantenedor"
+
+#: src/info.rs:68 src/info.rs:115
+msgid "Votes"
+msgstr "Votos"
+
+#: src/info.rs:69 src/info.rs:116
+msgid "Popularity"
+msgstr "Popularidad"
+
+#: src/info.rs:70 src/info.rs:117
+msgid "First Submitted"
+msgstr "Subido por primera vez"
+
+#: src/info.rs:71 src/info.rs:118
+msgid "Last Modified"
+msgstr "Modificado por última vez"
+
+#: src/info.rs:72 src/info.rs:120
+msgid "Out Of Date"
+msgstr "Desactualizado"
+
+#: src/info.rs:73 src/info.rs:125
+msgid "ID"
+msgstr "ID"
+
+#: src/info.rs:74 src/info.rs:126
+msgid "Package Base ID"
+msgstr "ID Base del Paquete"
+
+#: src/info.rs:75 src/info.rs:127
+msgid "Keywords"
+msgstr "Palabras clave"
+
+#: src/info.rs:76 src/info.rs:129
+msgid "Snapshot URL"
+msgstr "URL de Instantánea"
+
+#: src/info.rs:92
+msgid "No"
+msgstr "No"
+
+#: src/info.rs:112
+msgid "Optional Deps"
+msgstr "Dependencias Opcionales"
+
+#: src/install.rs:131 src/install.rs:545 src/install.rs:944
+msgid "no architecture"
+msgstr "sin arquitectura"
+
+#: src/install.rs:138
+msgid "can't build package as root"
+msgstr "no se puede construir paquete como root"
+
+#: src/install.rs:142
+msgid "failed to parse srcinfo generated by makepkg"
+msgstr "error al parsear srcinfo generado por makepkg"
+
+#: src/install.rs:171 src/install.rs:442
+msgid "Resolving dependencies..."
+msgstr "Resolviendo dependencias..."
+
+#: src/install.rs:220 src/install.rs:225
+msgid "failed to download sources"
+msgstr "error al descargar fuentes"
+
+#: src/install.rs:230 src/install.rs:246 src/install.rs:254
+msgid "failed to build"
+msgstr "error al construir"
+
+#: src/install.rs:233
+msgid "parsing pkg list..."
+msgstr "parseando lista pkg..."
+
+#: src/install.rs:261 src/install.rs:1462
+msgid "{}-{} is up to date -- skipping build"
+msgstr "{}-{} esta actualizado -- saltando construcción"
+
+#: src/install.rs:333 src/query.rs:50 src/lib.rs:84
+msgid "error:"
+msgstr "error:"
+
+#: src/install.rs:334
+msgid "could not get news"
+msgstr "no se pudieron obtener noticias"
```

Comment:
```suggestion
msgstr "no se pudo obtener noticias"
```

Third person singular because a tool is doing the work, not a group of tools/persons/etc.

Reply:
This and others. Is the conversation elsewhere?

Reply:
https://github.com/Morganamilo/paru/pull/435#discussion_r660050859